targets/TARGET_STM/lp_ticker.c

```diff
@@ -263,7 +258,6 @@ uint32_t lp_ticker_read(void)
 
 void lp_ticker_set_interrupt(timestamp_t timestamp)
 {
-    lp_ticker_disable_interrupt();
```

Comment:
Do you have any details on how this fixes the test in #8129?

Reply:
yes, I should add the details you have added in #8129